app/main.py

```
@@ -1,34 +1,61 @@
+from typing import List, Tuple
+
+
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self, row: int, column: int, is_alive: bool = True) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
 
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
-        # Create decks and save them to a list `self.decks`
-        pass
+    def __init__(self, start: Tuple[int, int],
+                 end: Tuple[int, int], is_drowned: bool = False) -> None:
+        self.decks = []
+        if start[0] == end[0]:
+            for col in range(start[1], end[1] + 1):
+                self.decks.append(Deck(start[0], col))
+        elif start[1] == end[1]:
+            for row in range(start[0], end[0] + 1):
+                self.decks.append(Deck(row, start[1]))
+        self.is_drowned = is_drowned
 
-    def get_deck(self, row, column):
-        # Find the corresponding deck in the list
-        pass
+    def get_deck(self, row: int, column: int) -> Deck:
+        for deck in self.decks:
+            if deck.row == row and deck.column == column:
+                return deck
+        return None
```

The `get_deck` method returns `None` if no deck is found at the specified coordinates. It would be better to raise an exception or handle this case more explicitly to avoid potential errors when `None` is returned.